101-automation/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
format the JSON, e.g. SHIFT+ALT+F in VS Code

Reply:
Done